mautic/mautic
# Fix for dropdowns in table-responsive using JS

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | n
| Related user documentation PR URL | na
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | #2458
| BC breaks? | n
| Deprecations? | n

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

The dropdowns on the last rows in table-responsive classes were cut off due to a "won't fix" Bootstrap issue. This uses JS to implement a fix by calculating and adding a padding.

#### Steps to test this PR:
1. Go to form results
2. Click the dropdown on the left of the last row
3. The delete button should be visible

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Same as above but the delete button will be hidden
